src/main/java/techcourse/myblog/dto/ArticleDto.java

```diff
@@ -0,0 +1,17 @@
+package techcourse.myblog.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import techcourse.myblog.domain.Article;
+
+@Getter
+@Setter
+public class ArticleDto {
+    private String title;
+    private String contents;
+    private String coverUrl;
+
+    public Article toEntity() {
+        return new Article(title, contents, coverUrl);
```

Comment:
https://stackoverflow.com/questions/21554977/should-services-always-return-dtos-or-can-they-also-return-domain-models